index.html

```diff
@@ -124,7 +124,7 @@
                 <li>
                     <div class="navbar-form" role="search" id="searchtool">
                         <div class="form-group input-group">
-                            <input type="text" class="form-control" placeholder="Rechercher" i18n="navbar.search.placeholder" id="searchfield">
+                            <input type="text" class="form-control" placeholder="Rechercher un lieu" i18n="navbar.search.placeholder" id="searchfield">
```

Comment:
All customizations have to be in custom component not in core mviewer

Reply:
Ok, I can remove this. I thought it would be more intuitive to illustrate what the user is searching for in general (but especially when the extension is used).

Reply:
>  I thought it would be more intuitive to illustrate what the user is searching

This search item could be use to find an address , a feature with FUSE  or ELS. Really usefull 👌

https://mviewerdoc.readthedocs.io/fr/latest/doc_tech/config_fuse.html

Reply:
Ok I removed the commit editing the search placeholder and renamed the layerfilter placeholder into "Filtrer les couches" for better distinction

Reply:
Seems better ;) Thanks !